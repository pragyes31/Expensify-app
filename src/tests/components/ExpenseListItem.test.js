import React from 'react';
import {shallow} from 'enzyme';
import expenses from '../fixtures/expenses'
import ExpenseListItem from '../../components/ExpenseListItem'

test('should render one list item', () => {
    const wrapper = shallow(<ExpenseListItem expense={expenses[2]}/>)
    expect(wrapper).toMatchSnapshot();
})