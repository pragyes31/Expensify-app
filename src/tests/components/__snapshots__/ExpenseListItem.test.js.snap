// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render one list item 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/undefined"
  >
    <h3 />
  </Link>
  <p>
     - 
  </p>
</div>
`;
